### DISCUSSION TITLE
Glider library recommendations?

### DISCUSSION THREAD

**Discussion by aaronadamsCA** [system-ui/theme-ui#1903] [Q&A]
Can anyone recommend a React glider/swiper library that can be made to work well with Theme UI?

A lot of our interfaces use side-scrollers. In a perfect world a library would be lightweight _and_ easy to initially implement with its built-in CSS _and_ then simple to eventually replace with our own theming.

Doing a lot of our own investigations, but wondering if anyone's got a personal recommendation. Thanks!

Keywords: `glider-js`, `react-slider`, `react-slick`, carousel, gallery, glider, slider, swiper

**Comment by sergeylukin** [comment]
I recommend https://www.npmjs.com/package/react-spring-carousel-js
